src/CodeGen_Hexagon.cpp

```diff
@@ -531,6 +534,8 @@ void CodeGen_Hexagon::init_module() {
 
         // Broadcasts
         { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatw), u32v1,  "splat.w", {u32} },
+        { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatb), u8v1,   "splat.b", {u8}  },   // v62 - Broadcasts
```

Comment:
OK, I think this is what we should do:

1. Revert changes to splat functions
2. Add 3 new intrinsics to this list: 

    { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatb), u8v1,   "splat_v62.b", {u8} },
    { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplath), u16v1,   "splat_v62.h", {u16} },
    { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatw), u32v1,  "splat_v62.w", {u32} },

3. Change the broadcast visitor to use `splat_v62` on v62 targets, and `splat` otherwise (https://github.com/halide/Halide/blob/ea3a49913527b6679ff65537d1b4539f4deb93cd/src/CodeGen_Hexagon.cpp#L1582)